explorer/app_settings.py

```diff
@@ -17,7 +18,13 @@
     ('csv', 'explorer.exporters.CSVExporter'),
     ('excel', 'explorer.exporters.ExcelExporter'),
     ('json', 'explorer.exporters.JSONExporter'),
+
 ])
+if sys.version_info[0] < 3:
```

Comment:
Since this stuff is in the optional requirements, I think we should do a try...except around importing one of the dependencies as well as checking the Python version before importing it. Otherwise we leave open the possibility of runtime errors when folks select the PDF download and the deps aren't installed.